stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/EditReactionsView.kt

```diff
@@ -0,0 +1,87 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.ui.utils.UiUtils
+
+public class EditReactionsView : ReactionsView {
+    private val bubblePaint = Paint().apply {
+        style = Paint.Style.FILL
+        color = Color.WHITE
+    }
+
+    public constructor(context: Context) : super(context) {
+        init(context, null)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init(context, attrs)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init(context, attrs)
+    }
+
+    private fun init(context: Context, attrs: AttributeSet?) {
+        setStyle(EditReactionsViewStyle(context, attrs))
+    }
+
+    override fun createReactionItems(message: Message, isMyMessage: Boolean): List<ReactionsAdapter.ReactionItem> {
+        return UiUtils.getReactionTypes().keys.map { reactionType ->
+            message.ownReactions
+                .any { it.type == reactionType }
+                .let { ReactionsAdapter.ReactionItem(Reaction(type = reactionType), it) }
```

Comment:
Nit: Would be nice to name `it` on this line to clarify what it means at this point